packages/core/core/src/requests/WriteBundlesRequest.js

```diff
@@ -83,6 +79,20 @@ async function run({input, api, farm, options}: RunInput) {
     return true;
   });
 
+  let ref;
+  let dispose;
+
+  // Create shared reference in WorkerFarm if we need to change multiple bundles and if we
+  // can't skip the subrequests for all the bundles
+  if (
```

Comment:
Not sure if I got this logic quite right.. are there any other cases to look for here?

Reply:
I'm not sure either, but I'm confused that the comment says "can't skip all the bundles = at least 1 bundle can't be skipped" but then the if condition checks "there are at least 2 bundles which can't be skipped".